sprints/dashboard/automation.py

```diff
@@ -231,6 +231,10 @@ def check_issue_injected(conn: CustomJira, issue: Issue) -> bool:
         "%Y-%m-%dT%H:%M:%S"
     )
 
+    # If a ticket has been created with a specific "sprint" value, this will not be indicated in its changelog.
+    if getattr(issue.fields, conn.issue_fields[settings.JIRA_FIELDS_CREATED]) >= cutoff_date:
```

Comment:
This expression has `str` type:
```python
getattr(issue.fields, conn.issue_fields[settings.JIRA_FIELDS_CREATED])
```

As well as the following:
```python
get_specific_day_of_sprint(settings.SPRINT_ASYNC_TICKET_CREATION_CUTOFF_DAY).strftime(
    "%Y-%m-%dT%H:%M:%S"
)
```

So, basically we compare two strings here. I think we have to cast both to `datetime` objects.